ui/common/src/main/res/values/strings.xml

```diff
@@ -868,6 +868,8 @@
   <string name="promotions_perks_button">Perks</string>
   <string name="promotions_vouhers_button">Vouchers</string>
   <string name="promotions_tooltip">Earn free AppCoins Credits with our promotions!</string>
+  <string name="promotions_bank_transfer_title">Easy Bank Transfer</string>
+  <string name="promotions_bank_transfer_body">Get %s%% extra bonus in all top-ups made with Easy Bank Transfer.</string>
```

Comment:
same issue as above

Reply:
I prefer to leave this one in title case to be consistent with our notifications standards. Usually our notifications titles are in title case.